lib/virt_autotest/utils.pm

```diff
@@ -166,6 +173,34 @@ sub is_monolithic_libvirtd {
     return 0;
 }
 
+# Restart libvirt daemon
+sub restart_libvirtd {
+    if (is_sle('<12')) {
+        assert_script_run('rclibvirtd restart', 180);
+    }
+    elsif (is_alp) {
+        my $_libvirtd_pid = script_output(q@ps -ef |grep [l]ibvirtd | gawk '{print $2;}'@);
+        my $_libvirtd_cmd = script_output("ps -o command $_libvirtd_pid | tail -1");
+        assert_script_run("kill -9 $_libvirtd_pid");
+        assert_script_run("$_libvirtd_cmd");
```

Comment:
It seems that there will be multiple new services for container based kvm server, for example, container-virtqemud.service. Anyway this alp part needs to be refined later.

Reply:
Indeed. Supposed that [poo#127652](https://progress.opensuse.org/issues/127652) will be handled the enhancement for alp part.